sample_files/README.md

```diff
@@ -0,0 +1,4 @@
+
+This directory is only intended for template files that are used for automatic driver initialization.
```

Comment:
Perhaps then s/sample_files/examples/ ? It's common to have an example directory in the root directory of a project.

Reply:
I'm not really sure what you mean, but I see that template_* is a better name. I changed all sample_* names to template_*.
We can discuss how to structure this further on Wed.

Reply:
Let's put the file in one of the recommended directories: https://api.csswg.org/bikeshed/?force=1&url=https://raw.githubusercontent.com/vector-of-bool/pitchfork/develop/data/spec.bs#intro.files